appwrite/appwrite
# Set certbot config directory

## What does this PR do?

Sets the `certbot` config directory via a command flag, so we can more usefully leverage `certbot` to handle certificate renewals more intelligently.

## Test Plan

- Deploy to a public server, with a domain configured
- Obtain SSL certificate
- Restart Appwrite stack
- Check `docker logs appwrite-worker-certificates` for `Exception: Renew isn't required` 

## Related PRs and Issues

#1025 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.
